### DISCUSSION TITLE
Expand sections on the new helper-scripts.com/scripts

### DISCUSSION THREAD

**Discussion by jcastro** [tteck/Proxmox#2927] [Ideas]
Hi!
I appreciate the recent improvements. I have a suggestion about the search function.

When looking for scripts, it would be helpful if the sidebar sections expanded automatically as you start searching. This way, we can quickly view them without needing to click on each one individually!

<img width="363" alt="image" src="https://github.com/tteck/Proxmox/assets/190036/11337af8-0e07-465d-815f-ca9a40ecec88">

**Comment by tteck** [comment]
ok

**Comment by tteck** [comment]
Actually, I prefer the current view. I'm looking for a VM so

![Screenshot 2024-05-07 11 09 31 AM](https://github.com/tteck/Proxmox/assets/21342670/2c7d3eb0-9d88-47c0-a895-e93d54f47bad)

It is showing me what categories have a virtual machine in them. Displaying categories is my preferred method. 

nginx:

![Screenshot 2024-05-07 11 17 31 AM](https://github.com/tteck/Proxmox/assets/21342670/942b917a-80d4-4cba-8bf8-d2170183cbdf)

Sorry for the extra clicks.

**Comment by jcastro** [comment]
You can still view the category; it's just expanded from the beginning. 

I'm assuming users typically will search for a specific service/brand name rather than for the type (e.g., VM, LXC).

**Reply by tteck** [reply]
Apologies for any confusion, but as mentioned earlier, I prefer displaying only categories. I find it creates a tidier presentation.

**Comment by jcastro** [comment]
Got it, thanks for considering it anyway!
